quiz_game.py

```diff
@@ -1,40 +1,67 @@
-print("Welcome to my computer quiz!")
+print('Wellcome to my Computer game! ')
 
-playing = input("Do you want to play? ")
-
-if playing.lower() != "yes":
+playing = input('Do u Want to play!')
+if playing != 'yes':

```

Comment:
There's a possibility that Yes, YES, yES etc is being typed into. So, .lower() is preferred by me. It's just a suggestion.